src/type_checker/resources/primitives/string.py

```diff
@@ -0,0 +1,15 @@
+class str:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'str'](https://app.codacy.com/app/JSAbrahams/mamba/pullRequest?prid=4163821)